elytron/src/test/resources/org/wildfly/extension/elytron/filesystem-realm-5/M/MF2HI4TJMJ2XIZKVONSXE.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+<identity xmlns="urn:elytron:1.0">
```

Comment:
Just to check, is there a way to generate these files dynamically as part of the test? If not, it's ok to add them.

Reply:
Should the schema version here be updated?

Reply:
Right for the files that use only encryption the schema version should be `urn:elytron:1.0` but for integrity enabled it should be `urn:elytron:identity:1.1`